akka-docs-dev/rst/java/stream-integrations.rst

```diff
@@ -429,4 +431,7 @@ by using the Subscriber-:class:`Source`:
 
 .. includecode:: ../../../akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/ReactiveStreamsDocTest.java#sink-subscriber
 
+It is also possible to use :class:`Processor` interfaces directly with both :class:`Source` and :class:`Flow` by
```

Comment:
This should go to the top of the section then, this is not unique to Processors.